I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p build
+cd build
+# this line seems to prevent compilation; removed for testing:
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
+    -DCGAL_test_cpp_version_RUN_RES=0 \
+    -DCGAL_test_cpp_version_RUN_RES__TRYRUN_OUTPUT="$(c++ -xc++ - -E -dM < /dev/null | grep -oP '__cplusplus \K.*')" \
```

Comment:
This is a weird line, I wonder what it is about? It seems to extract the supported C++ version. But why is this necessary?

The `libcgal_julia` builder seems to be able to link with CGAL here without any special contortions. I wonder where the difference comes from?

Reply:
CMake was failing because they were running `try_run` for what is a compiler check (and `try_run` doesn't work at all in cross-compilation mode), so you must provide the output of the command (even though it isn't really used, as far as I can tell, so empty output might work as well).  See https://github.com/JuliaPackaging/Yggdrasil/pull/3832#discussion_r739722131

Reply:
Ah I see. Might I suggest to add that explanation in a comment? It might help people who want to edit this builder in the future (e.g. you own future self ;-) ). Just a suggestion, though (but I usually benefit from doing it when I remember to do it in "my" buulder)

Reply:
(It's pretty cool that cmake allows overriding the test like this, I must say (even though in general I am not a fan))